frontend/src/components/dashboard/aliases/AliasList.tsx

```diff
@@ -169,12 +169,12 @@ export const AliasList = (props: Props) => {
         </div>
         <button
           onClick={() => setStringFilterVisible(!stringFilterVisible)}
-          title={l10n.getString("banner-register-subdomain-button-search")}
+          title={l10n.getString("profile-filter-search-placeholder-2")}
```

Comment:
This is the button to toggle the mask search field. It re-used the subdomain search label since that happened to say "Search" as well, but the context is different (possibly someone translated that as "Search subdomain"). Since the tests looks for this button label, it would get confused which search button I needed. Thus, I replaced this by the existing relevant string that said "Search masks".

![image](https://user-images.githubusercontent.com/4251/173556210-a68b42f1-1cde-4836-9bdd-50eebc035f3b.png)

Reply:
🦅 Catch!